ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/data/PipelineConfigDao.kt

```diff
@@ -0,0 +1,24 @@
+package ai.nnstreamer.ml.inference.offloading.data
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import androidx.room.Update
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface PipelineConfigDao {
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
+    fun insert(pipelineConfig: PipelineConfig)
+
+    @Update
+    suspend fun update(pipelineConfig: PipelineConfig)
+
+    @Delete
+    suspend fun delete(pipelineConfig: PipelineConfig)
+
+    @Query("SELECT * from pipelineConfigs")
+    fun getConfig(): Flow<List<PipelineConfig>>
```

Comment:
```SELECT * from pipelineConfigs``` seems that getting all configs in the DB instead of getting a config

Reply:
Updated! Thank you :)